static/src/jspm-config.js

```diff
@@ -0,0 +1,135 @@
+System.config({
+  "baseURL": "/assets",
+  "transpiler": "babel",
+  "paths": {
+    "*": "javascripts/projects/*.js",
+    "es6/*": "javascripts/es6/*.js",
+    "bootstraps/*": "javascripts/bootstraps/*.js",
```

Comment:
Yes this is safe, and a perfectly ok workflow. These paths are kept in-tact by jspm. We used to manage local paths with the local `directories.lib` but removed that functionality as it was confusing and doing too much.